pkg/Library/OpenAI/Atoms/OpenAIImage.js

```diff
@@ -18,30 +18,34 @@ initialize(inputs, state, {service}) {
     n: 1
   });
 },
-shouldUpdate({enabled, prompt}) {
-  return enabled && prompt;
+shouldUpdate({/*enabled,*/ prompt}) {
```

Comment:
I think we need to be able to turn this off too.

Reply:
restored everywhere (for whatever reason at the moment i though `restart` is sufficient to avoid triggering... but that was wrong)